src/cogs/help.py

```diff
@@ -70,13 +83,16 @@ def get_cog_help(cog: commands.Cog) -> discord.Embed:
             # In python 3.10 this should be a match.
             if isinstance(cmd_or_group, commands.Group):
                 out.add_field(
-                    name=cmd_or_group.qualified_name,
+                    name=f"```{cmd_or_group.qualified_name}```",
                     value=f"{count(cmd_or_group.walk_commands())} commands.",
                 )
                 continue
             out.add_field(
                 name=command_usage(cmd_or_group),
-                value=(cmd_or_group.help if cmd_or_group.help else "No help given"),
+                value="```"
+                + (cmd_or_group.help if cmd_or_group.help else "No help given")
+                + "```",
+                inline=False,
```

Comment:
Question: I see this pattern a few times. Is it worth making a codeblock() util function that just does this in favor of readability?